### DISCUSSION TITLE
MPC Connection to Torus Failed

### DISCUSSION THREAD

**Discussion by 0xalank** [Web3Auth/Web3Auth#887] [Help]
Using the demo code from https://github.com/torusresearch/w3a-mpc-demo/blob/master/src/App.tsx inside of a React Chrome Extension. Whitelist label in the dashboard is set to `chrome-extension://<extension-id>`

Able to correct pop up the modal and redirect to this screen:
![Screen Shot 2022-11-16 at 11 14 53 AM](https://user-images.githubusercontent.com/14866363/202248216-127e02ca-af58-4670-b001-cdb6a270d843.png)

Logs on the unable to connect screen:
![Screen Shot 2022-11-16 at 11 04 22 AM](https://user-images.githubusercontent.com/14866363/202247791-424cbb1c-a245-422c-a2d3-e514c3902d05.png)
![Screen Shot 2022-11-16 at 11 04 38 AM](https://user-images.githubusercontent.com/14866363/202247799-621db43f-a864-4ac0-84ab-5ba1dd210916.png)

If it means anything the Demo on the main site is down as well for me. https://demo-mpc.web3auth.io/

**Comment by chaitanyapotti** [comment]
The demo is fixed now. It was using outdated endpoints
Thanks for reporting

**Reply by 0xalank** [reply]
What is the new endpoint? Using MPC Alpha for a demo of my own.
